### PR TITLE
Happy New Year - Fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- removes outer jQuery shorthand
- fixes url checkup
- replaces .error with .fail on ajax requests

please use ``?w=1`` for review: https://github.com/FinalAngel/django-cms/pull/113?w=1

### CODE REVIEW

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 1]
```diff
@@ -14,615 +14,615 @@ var CMS = window.CMS || {};
...
+        _setAjaxPost: function _setAjaxPost(trigger) {
+            var that = this;
+
+            this.ui.container.on(this.click, trigger, function (e) {
+                e.preventDefault();
+                $.ajax({
+                    method: 'post',
+                    url: $(this).attr('href')
+                }).done(function () {
+                    CMS.API.Helpers.reloadBrowser();
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: this is not an equivalent change, is that ok?

**Comment by vxsx** [comment]
overall LGTM

File: cms/static/cms/js/modules/cms.pagetree.js (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.pagetree.js]: yes, reload should always be called with the helper, in this case I just forgot to change it

Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: cool

**Comment by FinalAngel** [comment]
Will add the missing pieces also to this PR:

- use correct setSettings and getSettings
- remove remaining todos

so the code is ready for design implementation

**Comment by vxsx** [comment]
LGTM

**Pull request merged**